### DISCUSSION TITLE
AutoComplete DebounceMilliseconds?

### DISCUSSION THREAD

**Discussion by michaelcsikos** [ant-design-blazor/ant-design-blazor#4324] [Ideas]
I need to be able to set `DebounceMilliseconds` on `AutoComplete`. Is there a way?

**Comment by michaelcsikos** [comment]
@ElderJames, any ideas?

**Comment by ElderJames** [comment]
Hello @michaelcsikos , the AutoComplete has the DebounceMilliseconds parameter, does it not work?

**Reply by michaelcsikos** [reply]
I see it now, thanks. Maybe it was added in a recent update.
